docs/references.md

```diff
@@ -160,7 +160,7 @@ When run <FUNCTION | COMMAND> [ARGUMENTS...]
 
 This is primarily designed for external command calls.
 The external command does not have to be a shell script.
-Even shell scripts are processed as external commands (respect shebang), so it not covered by coverage.
+Even shell scripts are processed as external commands (respect shebang), so it is not covered by coverage.
```

Comment:
```suggestion
Even shell scripts are processed as external commands (and thus respecting shebang), so they are not covered by coverage.
```

Reply:
Thank you for your suggestions and sorry for the confusion.

Is the meaning conveyed with this sentence?

> Even shell scripts are executed as external commands according to the shebang, so they are not covered by coverage.

Reply:
Yes, I think it is better. 
Thank you!